### PR TITLE
[4.0] Group action toolbar buttons usernotes

### PR DESCRIPTION
**Pull request by chmst**
### Summary of Changes
This PR continues the grouping of action buttons in the backend (see #22832, #22393, #23337 and #23336, #23374, #23368). This time the usernotes.

### Testing Instructions
Go to the user notes in the backend and check the toolbar. There should be a "Change status" dropdown now which activates when you select a user note.

### CODE REVIEW

**Comment by chmst** [comment]
While testing this you will note that changing states throws an error. https://issues.joomla.org/tracker/joomla-cms/23384<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23380">issues.joomla.org/tracker/joomla-cms/23380</a>.</sub>

File: administrator/components/com_users/View/Notes/HtmlView.php [hunk 1]
```diff
@@ -135,40 +136,49 @@ protected function addToolbar()
 
 		ToolbarHelper::title(Text::_('COM_USERS_VIEW_NOTES_TITLE'), 'users user');
 
+		// Get the toolbar object instance
+		$toolbar = Toolbar::getInstance('toolbar');
+
 		if ($canDo->get('core.create'))
 		{
-			ToolbarHelper::addNew('note.add');
+			$toolbar->addNew('user.add');
```
Comment (Hackwar) [inline administrator/components/com_users/View/Notes/HtmlView.php]: Shouldn't this still read "note.add" and not "user.add"?

**Comment by Hackwar** [comment]
I have tested this item :white_check_mark: successfully on 577cc34b0f4723cbaa214521b2ffb0487a8a9735<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23380">issues.joomla.org/tracker/joomla-cms/23380</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**